test/pkg/github/pr.go

```diff
@@ -100,6 +102,7 @@ func PRCreate(ctx context.Context, cs *params.Run, ghcnx *ghprovider.Provider, o
 }
 
 func RunPullRequest(ctx context.Context, t *testing.T, label string, yamlFiles []string, webhook bool) (*params.Run, *ghprovider.Provider, options.E2E, string, string, int, string) {
+	time.Sleep(5 * time.Second)
```

Comment:
shouldn't this wait be in the cleanup function?

Reply:
i think i was doing this once, but we were running out of memory or etcd was freaking off when there was too many pods with memory constraint